src/yadism/esf/tmc.py

```diff
@@ -479,7 +537,67 @@ def _get_result_exact(self):
         return self._factor_shifted * F3out + self._factor_h3 * h3out
 
 
-ESFTMCmap = {"F2": ESFTMC_F2, "FL": ESFTMC_FL, "F3": ESFTMC_F3}
+class ESFTMC_g1(EvaluatedStructureFunctionTMC):
+    """
+    This function implements the actual formula for target mass corrections
+    for parity conserving polarized structure function g1, for the two kinds
+    described in the parent class :py:class:`EvaluatedStructureFunctionTMC`.
+
+    The formula in question can be found in :cite:`Accardi:2008pc`,
```

Comment:
Can you please quote the exact equation? This makes my review easier :upside_down_face:  I guess it is Eq. 10 of `:cite:tmc-g1-iranian`, yes?